stream_alert_cli/lookup_tables/handler.py

```diff
@@ -0,0 +1,187 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from stream_alert.shared.logger import get_logger
+from stream_alert.shared.lookup_tables.core import LookupTables
+from stream_alert_cli.utils import CLICommand, generate_subparser
+
+LOGGER = get_logger(__name__)
+
+
+class LookupTablesCommand(CLICommand):
+
+    description = 'Describe and manage your LookupTables'
+
+    @classmethod
+    def setup_subparser(cls, subparser):
+        """Add subparser for LookupTables"""
+        lookup_tables_subparsers = subparser.add_subparsers()
+
+        cls._setup_describe_tables_subparser(lookup_tables_subparsers)
+        cls._setup_get_subparser(lookup_tables_subparsers)
+        cls._setup_set_subparser(lookup_tables_subparsers)
+
+    @classmethod
+    def _setup_describe_tables_subparser(cls, subparsers):
+        generate_subparser(
+            subparsers,
+            'describe-tables',
+            description='Show tables',
+            subcommand=True
+        )
+
+    @classmethod
+    def _setup_get_subparser(cls, subparsers):
+        """
+        Get subcommand:
+
+        $ ./manage.py lookup-tables get -t [table] -k [key]
+        """
+        get_parser = generate_subparser(
+            subparsers,
+            'get',
+            description='Validate defined log schemas using integration test files',
+            subcommand=True
+        )
+
+        get_parser.add_argument(
+            '-t',
+            '--table',
+            help='Name of the LookupTable',
+            required=True,
+            default=None
+        )
+
+        get_parser.add_argument(
+            '-k',
+            '--key',
+            help='Key to fetch on the LookupTable',
+            required=True,
+            default=None
+        )
+
+    @classmethod
+    def _setup_set_subparser(cls, subparsers):
+        """
+        Set subcommand:
+
+        $ ./manage.py lookup-tables set -t [table] -k [key] -v [value]
```

Comment:
note - you can use an `epilog` to actually help with providing a good example to the user